app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,21 @@
+class TagsController < ApplicationController
+  TAGGABLE_MODELS = %w(Group Resource).freeze
```

Comment:
We'd also like Lists to be taggable. Should we add that in a later PR once the List models are all set up?